clients/windows/src/window.rs

```diff
@@ -294,30 +277,43 @@ fn handle_message(hwnd: HWND, message: Message) -> bool {
 
                     match message {
                         MessageAppDep::KeyDown { key } | MessageAppDep::KeyUp { key } => {
+                            unsafe { InvalidateRect(hwnd, None, false) };
                             input::key::handle(app, message, key, modifiers)
                         }
                         MessageAppDep::LButtonDown { pos }
                         | MessageAppDep::LButtonUp { pos }
                         | MessageAppDep::RButtonDown { pos }
                         | MessageAppDep::RButtonUp { pos }
                         | MessageAppDep::MouseMove { pos } => {
+                            unsafe { InvalidateRect(hwnd, None, false) };
                             input::mouse::handle(app, message, pos, modifiers, window.dpi_scale)
                         }
                         MessageAppDep::PointerDown { pointer_id }
                         | MessageAppDep::PointerUpdate { pointer_id }
-                        | MessageAppDep::PointerUp { pointer_id } => input::pointer::handle(
-                            app,
-                            hwnd,
-                            modifiers,
-                            window.dpi_scale,
-                            pointer_id,
-                        ),
+                        | MessageAppDep::PointerUp { pointer_id } => {
+                            unsafe { InvalidateRect(hwnd, None, false) };
+                            input::pointer::handle(
+                                app,
+                                hwnd,
+                                modifiers,
+                                window.dpi_scale,
+                                pointer_id,
+                            )
+                        }
                         MessageAppDep::MouseWheel { delta }
                         | MessageAppDep::MouseHWheel { delta } => {
+                            unsafe { InvalidateRect(hwnd, None, false) };
                             input::mouse::handle_wheel(app, message, delta)
                         }
                         MessageAppDep::SetCursor => output::cursor::handle(),
                         MessageAppDep::Paint => {
+                            // "you'll find that your UI thread starts to burn 100% cpu core and your WM_PAINT handler
+                            // getting called over and over again... WM_PAINT is generated as long as the window has a
+                            // dirty rectangle, created by an InvalidateRect() call by either the window manager or
+                            // your program explicitly calling it. BeginPaint() clears it."
+                            // https://stackoverflow.com/questions/5841299/difference-between-getdc-and-beginpaint
```

Comment:
This is the magic ingredient I was missing (it's a code comment instead of a review comment! growth 🌱)